cypress/integration/iframe.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { IframePage } from "../page/index.ts";
+
+let iframePage: IframePage;
+
+describe("Iframe Interaction", () => {
+  before(() => {
+    iframePage = new IframePage();
+    Cypress.on("uncaught:exception", () => false);
+    iframePage.visit();
+  });
+
+  it("Interacting with the Iframe ", () => {
+    iframePage.getFrame().find("h1").should("have.text", "HTML Tutorial");
+    iframePage.goToCssPageInFrame();
+
```

Comment:
You should wait for the iframe to load. See: https://www.npmjs.com/package/cypress-iframe#frameloaded